spec/models/user_spec.rb

```diff
@@ -224,4 +224,32 @@
       expect(user.decorate).to be_a(UserDecorator)
     end
   end
+
+  describe '#password_reset_profile' do
+    let(:user) { create(:user) }
+    subject(:password_reset_profile) { user.password_reset_profile }
```

Comment:
it looks like we're not using `subject` in the rest of this spec file. I myself like the style of explicitly testing a var rather than using `subject` / `it` -- change for consistency?